docker/README.md

```diff
@@ -238,7 +238,7 @@ If you use one-off domains, you'll have to re-install WordPress and re-connect J
 
 If you're developing Jetpack often you'll want to reserve a domain you can keep using.
 
-If you are an Automattician, sign up on Ngrok.com using your a8c Google account; you'll be automattically added to the Automattic team. That will enable you to re-use domains, reserve your custom domains and reserve TCP ports.
```

Comment:
👎

Reply:
@lezama - Not sure what you mean here?

Reply:
😅 I am kidding, showing disappointment about not using `automattically`

Reply:
I love that we are adding spell checking ❤️